### ISSUE TITLE
Mostrando número USP publicamente ao visualizar reserva

### ISSUE THREAD

**Issue by kawanks** [uspdev/salas#181]
Nas rotas `/reservas/{reserva_id}`  onde é possível visualizar a reservas, é mostrado o número USP da pessoa que cadastrou a reserva de forma pública. Seria melhor não deixar esta informação pública ou o número USP não é um dado sensível?

**Comment by alecostaweb** [comment]
Entendo que o número USP não deva aparecer de forma pública ou anônima.

**Comment by thiagogomesverissimo** [comment]
Eu não sei... Tem vários sistemas por ai que mostram o número USP. Mas melhor deixar só para logados apenas. O nome da pessoa pode mostrar normalmente, correto?

**Comment by alecostaweb** [comment]
Certo. O nome tem que mostrar, do contrário o controlador de acesso (sem logar é mais prático pra ele) fica sem saber responder aos alunos perdidos (geralmente, perguntam onde é a aula do Fulano?)

**Comment by thiagogomesverissimo** [comment]
Ficamos com o nome.  O Número USP realmente não é necessário.
